appinventor/components/src/com/google/appinventor/components/scripts/ComponentDescriptorGenerator.java

```diff
@@ -46,6 +46,7 @@
  *   }*,
  *   "properties": [
  *     { "name": "PROPERTY-NAME",
+ *        "description": "DESCRIPTION",  //[suvam, 2019/03/25] Added description field to json
```

Comment:
Please remove the comment within the comment. I think it's clear from the other comment further down when and why this was added. Removing the comment keeps the JSON template clean.

Reply:
It's done now